pkg/app/piped/planner/kubernetes/kubernetes.go

```diff
@@ -125,6 +125,14 @@ func (p *Planner) Plan(ctx context.Context, in planner.Input) (out planner.Outpu
 		return
 	}
 
+	// Force to use pipeline when the alwaysUsePipeline field was configured.
+	if cfg.Planner.AlwaysUsePipeline {
```

Comment:
Should we add TODO or something in planner for other kinds of deployment we're supporting? Since this configuration is added as GenericDeploymentSpec config.

Reply:
I have checked all other kinds and we don't need to have any additional updates for them.

Reply:
On the second thought, let me add this part for them too. For good consistency.

Reply:
Yeh, that's what I worried about. Since this configuration may lead users to think like they have to add this line in their configuration for all kinds of applications.

Reply:
I have just applied this new field to all application kinds. PTAL.

Reply:
Thanks 👍